src/pathfinding_service/database.py

```diff
@@ -265,8 +265,11 @@ def pop_waiting_messages(
             """,
             [to_checksum_address(token_network_address), hex256(channel_id)],
         ):
-            message: PFSFeeUpdate = JSONSerializer.deserialize(row["message"])
-            message.timestamp = message.timestamp.replace(tzinfo=timezone.utc)
+            message: DeferableMessage = JSONSerializer.deserialize(row["message"])
+
+            if isinstance(message, PFSFeeUpdate):
+                message.timestamp = message.timestamp.replace(tzinfo=timezone.utc)
```

Comment:
so what I understand here is that you replace the timestamp on the PFSFeeUpdate message with the current timestamp, right?